src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -127,6 +208,24 @@ struct bdf
   }
 };
 
+struct id
```

Comment:
Uhh, `id` is very generic, what is it in this context?  Maybe be a little more descriptive like `pcie_id` ?